entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -164,29 +193,30 @@ class EntityService(
         values.filterValues {
             it[0] is Map<*, *>
         }
-        .mapValues {
-            expandValueAsMap(it.value)
-        }
-        .filter {
-            !(NGSILD_PROPERTIES_CORE_MEMBERS.plus(NGSILD_RELATIONSHIPS_CORE_MEMBERS)).contains(it.key) &&
+            .mapValues {

```

Comment:
I find weird this extra indent. I guest it is considered like a continuation indent but ...

Reply:
I would be more comfortable with something like:

```
values
    .map {
    }
    .filter {
    }
```

Reply:
Rather than

```
values.map {
}
    .filter {
    }
```

Reply:
I don't think continuation indent applies to braces... 

I think expected result is : 
```kotlin
values.filterValues {
    it[0] is Map<*, *>
}.mapValues {
    expandValueAsMap(it.value)
}.filter {
    !(NGSILD_PROPERTIES_CORE_MEMBERS.plus(NGSILD_RELATIONSHIPS_CORE_MEMBERS)).contains(it.key) &&
        isAttributeOfType(it.value, NGSILD_PROPERTY_TYPE)
}.forEach { entry ->
    if (!neo4jRepository.hasPropertyOfName(subject.id, entry.key))
        throw BadRequestDataException("Property ${entry.key.extractShortTypeFromExpanded()} does not exist")
    updatePropertyValues(subject.id, entry.key, entry.value)
}
```
But there were a newline placed in first line..
What do you think ?

Reply:
I'm fine with your expected result!

Reply:
Fixed in d6d00ee57a34597b8636634c9f94a7731810dcac